### PR TITLE
SEO-176062-Windows-forms-duplicate-titles-

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @MallikaRK ,
Have updated UG titles please review.
Thanks.

Regards,
Edith Abich.

### CODE REVIEW

File: WindowsForms/DICOM/Overview.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: About Windows Forms dicom control | Syncfusion
+title: Overview of Windows Forms Dicom control | Syncfusion
```
Comment (Yvone-Atieno) [inline WindowsForms/DICOM/Overview.md:3]: The first word should be in caps under Control. Check in all places.

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: WindowsForms/DICOM/Overview.md (hunk 1 above)
Comment (Edithabich) [reply WindowsForms/DICOM/Overview.md:3]: resolved

File: WindowsForms/TabControl/Overview.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: About Windows Forms TabControl control | Syncfusion
+title: Overview of Windows Forms TabControl | Syncfusion
```
Comment (Yvone-Atieno) [inline WindowsForms/TabControl/Overview.md:3]: Overview of Windows Forms TabControl Control | Syncfusion

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: WindowsForms/TabControl/Overview.md (hunk 2 above)
Comment (Edithabich) [reply WindowsForms/TabControl/Overview.md]: resolved

**Pull request closed without merging**